internal/controllers/ipaddressclaim.go

```diff
@@ -47,22 +52,59 @@ type NutanixProviderAdapter struct {
 	pcClientGetter   func(pcclient.CachedClientParams) (pcclient.Client, error)
 	secretInformer   coreinformers.SecretInformer
 	cmInformer       coreinformers.ConfigMapInformer
+	opts             reconcilerOptions
 }
 
 var _ ipamutil.ProviderAdapter = &NutanixProviderAdapter{}
 
+type reconcilerOptions struct {
+	concurrentReconciles           int
+	minRequeueTime, maxRequeueTime time.Duration
+}
+
+func DefaultReconcilerOptions() reconcilerOptions {
+	return reconcilerOptions{
+		concurrentReconciles: 10,
+		minRequeueTime:       500 * time.Millisecond,
+		maxRequeueTime:       1000 * time.Second,
+	}
+}
+
+func (o *reconcilerOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.IntVar(
+		&o.concurrentReconciles,
+		"concurrent-reconciles",
```

Comment:
nit: shouldn't this be max-concurrent-reconciles?

Reply:
Updated.